community/kernel/src/main/java/org/neo4j/kernel/impl/store/kvstore/AbstractKeyValueStore.java

```diff
@@ -280,27 +282,22 @@ protected Value defaultValue()
         @Override
         public boolean visit( ReadableBuffer key, ReadableBuffer value )
         {
-            return visitKeyValuePair( readKey( key ), value );
+            try
+            {
+                return visitKeyValuePair( readKey( key ), value );
+            }
+            catch ( UnknownKey e )
+            {
+                return visitUnknownKey( e, key, value );
+            }
```

Comment:
Expected rare case. In reality it is a failure case, that method just throws a different exception, but allows for tools (`DumpCountsStore`) to continue.